openfisca_france_dotations_locales/parameters/dotation_solidarite_rurale/bourg_centre/eligibilite/seuil_nombre_habitants_chef_lieu.yaml

```
@@ -0,0 +1,6 @@
+description: Nombre strictement maximal d'habitants de la commune selon population DGF \
+  pour l'attribution de la DSR fraction bourg-centre si la commune est chef lieu d'arrondissement.
+reference: https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070633&idArticle=LEGIARTI000023650523
+values:
+  2019-01-01:
```

D'après la référence législative, il s'agit de la Loi de Finances applicable en janvier 2017 : 
```suggestion
  2017-01-01:
```